src/displayapp/fonts/generate.py

```diff
@@ -35,10 +35,11 @@ def main():
     ap = argparse.ArgumentParser(description='auto generate LVGL font files from fonts')
     ap.add_argument('config', type=str, help='config file to use')
     ap.add_argument('-f', '--font', type=str, action='append', help='Choose specific fonts to generate (default: all)', default=[])
+    ap.add_argument('--lv-font-conv', type=str, help='Path to "lv_font_conf" executable (default: lv_font_conf)', default="lv_font_conv")
```

Comment:
don't bother with the "(default: lv_font_conf)" in the help, if there's a default argparse will do that for you.

Reply:
```suggestion
    ap.add_argument('--lv-font-conv', type=str, help='Path to "lv_font_conf" executable', default="lv_font_conv")
```